enn_ppo/enn_ppo/train.py

```diff
@@ -555,9 +555,17 @@ def init_process(xp_info: Any, backend: str = "gloo") -> None:
 
 
 def gradient_allreduce(model: Any) -> None:
```

Comment:
@vwxyzjn simple change that significantly improves performance

Reply:
Oh nice! Good to know. I am gonna give it a try!